src/components/AppContext/AppContext.ts

```diff
@@ -0,0 +1,20 @@
+import React, { createContext } from "react";
+
+interface CompendiumAssets {
+  category: "materials" | "monsters";
+  common_locations: string[];
+  cooking_effect: string;
+  description: string;
+  dlc: boolean;
+  drops?: string[];
+  hearts_recovered: number;
+  id: number;
+  image: string;
+  name: string;
+}
+
+interface AppContextShape {
+  data?: CompendiumAssets[];
+}
+
+export const AppContext: React.Context<AppContextShape> = createContext({});
```

Comment:
nit - should we default the context to [] instead of {} to mimic the AppContextShape?

Reply:
Was thinking this might hold other state in the future so we could add things to the overall state alongside `data`